modelscope/pipelines/audio/speaker_verification_pipeline.py

```diff
@@ -181,7 +181,7 @@ def forward(self, audio_in: Union[tuple, str, Any] = None) -> list:
         """Decoding
         """
         logger.info(
-            'Speaker Verification Processing: {0} ...'.format(audio_in))
+            'Speaker Verification Processing: {0} ...'.format(str(audio_in)[:500]))
```

Comment:
the following code looks better and url can be judged using str.startswith('http')
```python
    # 判断 audio_in 是否为文件路径或 URL
    if isinstance(audio_in, str):
        parsed_url = urlparse(audio_in)
        if os.path.isfile(audio_in) or (parsed_url.scheme and parsed_url.netloc):
            # 如果是文件路径或 URL，则输出 audio_in
            logger.info(f'Speaker Verification Processing: {audio_in} ...')
        else:
            # 如果不是文件路径或 URL，则输出数据的前500字节
             logger.info(f'Speaker Verification Processing: {str(audio_in)[:500]} ...')
    else:
        # 如果不是文件路径或 URL，tuple or bytes 则输出数据的前500字节
        logger.info(f'Speaker Verification Processing: {str(audio_in)[:500]} ...')

```

Reply:
👌